task/02-numbers-tasks.js

```diff
@@ -89,7 +101,14 @@ function getDistanceBetweenPoints(x1, y1, x2, y2) {
  *   5*x = 0         => 0
  */
 function getLinearEquationRoot(a, b) {
-    throw new Error('Not implemented');
+
+    let result = -b/a;
+    if(result===0){
```

Comment:
This task can be performed in one "return" expression without unnecessary checking